apps/dashboard/src/env.ts

```diff
@@ -12,14 +12,20 @@ export const env = createEnvironment(
     NEXT_PUBLIC_ORIGIN: variable,
     NEXT_PUBLIC_RPC_HOST: variable,
     NEXT_PUBLIC_SENTRY_DSN: variable.optional(),
+    S3_UPLOAD_ENABLED: variable,
```

Comment:
Would this not be a more tidy way to do it? Doesn't really matter at all though
```suggestion
    S3_UPLOAD_ENABLED: variable.default(false),
```

Reply:
nice feedback, I like this way better